5thColumn/vee
# Basic trees

Apologies for the size of this PR. A lot of the logic here was originally hardcoded in an external project - this is the earliest state of refactoring that left me at least mildly happy.

I don't have a lot of experience with high-level javascript architecture. Tips around this sort of thing would be extremely welcome.

To test:
After starting, go to the root page (http://localhost:8080).
This will automatically load data from a pre-determined report (ID 393).
Play around with what's there. Keep devtools open and make sure console isn't throwing any errors.

Notes:
The control bar is there for the visual but is not functional - that's coming way down the road. So the play/pause buttons, progress bar, and flavor text aren't supposed to do anything yet or look pretty.
